collector.go

```diff
@@ -228,6 +228,8 @@ func pduValueAsString(pdu *gosnmp.SnmpPDU, typ string) string {
 		return strconv.FormatUint(uint64(pdu.Value.(uint)), 10)
 	case int64:
```

Comment:
Are you sure we want to do that?  What if we change the upstream snmp and this becomes a valid case?